ginga/BaseImage.py

```diff
@@ -391,9 +391,8 @@ def get_shape_mask(self, shape_obj):
         Return full mask where True marks pixels within the given shape.
         """
         wd, ht = self.get_size()
-        yi = np.mgrid[:ht].reshape(-1, 1)
-        xi = np.mgrid[:wd].reshape(1, -1)
-        pts = np.asarray((xi, yi)).T
+        xi, yi = np.meshgrid(range(0, wd), range(0, ht))
+        pts = np.array((xi, yi)).T.reshape(-1, 2)
```

Comment:
My hunch is that this new format of `pts` is not compatible with `shape_obj.contains_pts`.